src/views/ScopesetExpander/index.jsx

```diff
@@ -10,15 +10,29 @@ import Tooltip from '@material-ui/core/Tooltip';
 import List from '@material-ui/core/List';
```

Comment:
The scopesets expander & compare are two different tools which should live in their own views. Change required:
* Remove `/expansion` and replace with `/scopesets/compare` and `/scopesets/expand`.
* Add "Scopesets" as a parent menu in the left navigation drawer and include "Compare" and "Expand" as children to Scopesets.

Let me know if something is confusing and I will try to help :)

Reply:
@helfi92, What icon should I use besides "Scopesets" in the left navigation drawer? :confused:

Reply:
Feel free to choose from https://github.com/levrik/mdi-react.